starter-code/iteration-3/main.ts

```diff
@@ -1,9 +1,19 @@
 // Add the reference to the interface
-
+import {
+  TodoItemInterface, TodoListInterface
+} from "./interfaces";
 // Create class TodoItem that implements the corresponding interface
+class TodoItem implements TodoItemInterface {
+  constructor(public title:string){
 
+  };
+  status:boolean;
+  updatedAt:Date;
+};
 // Create class TodoList that implements the corresponding interface
```

Comment:
Era algo así:

// Create class TodoItem that implements the corresponding interface
class TodoItem implements TodoItemInterface {
  title: string   = "";
  status: boolean = false;
  updatedAt: Date;

  constructor(title: string = "New task") {
    this.title     = title;
    this.updatedAt = new Date();
  }

  toggleStatus() {
    this.status   != this.status;
    this.updatedAt = new Date();
  }
}